velox/connectors/hive/storage_adapters/abfs/AbfsConfig.cpp

```diff
@@ -39,30 +41,118 @@ AbfsConfig::AbfsConfig(
   fileSystem_ = file.substr(0, firstAt);
   auto firstSep = file.find_first_of("/");
   filePath_ = file.substr(firstSep + 1);
+  accountNameWithSuffix_ = file.substr(firstAt + 1, firstSep - firstAt - 1);
 
-  auto accountNameWithSuffix = file.substr(firstAt + 1, firstSep - firstAt - 1);
-  auto firstDot = accountNameWithSuffix.find_first_of(".");
-  auto accountName = accountNameWithSuffix.substr(0, firstDot);
-  auto endpointSuffix = accountNameWithSuffix.substr(firstDot + 5);
-  auto credKey = fmt::format("fs.azure.account.key.{}", accountNameWithSuffix);
-  std::stringstream ss;
-  ss << "DefaultEndpointsProtocol=" << (isHttps ? "https" : "http");
-  ss << ";AccountName=" << accountName;
-
-  if (config.valueExists(credKey)) {
+  auto authTypeKey =
+      fmt::format("{}.{}", kAzureAccountAuthType, accountNameWithSuffix_);
+  authType_ = "SharedKey";
+  if (config.valueExists(authTypeKey)) {
+    authType_ = config.get<std::string>(authTypeKey).value();
```

Comment:
Is the value of `authTypeKey` case-sensitive?

Reply:
Yes.